### ISSUE TITLE
Title should respect theme and alignment

### ISSUE THREAD

**Issue by chemelli74** [custom-components/uilogs#9]
Title should be better aligned and respect the theme (font, underlining):

Uilogs top bar:

![image](https://user-images.githubusercontent.com/57354320/80861717-91e2b100-8c70-11ea-87ca-d66e3c3da188.png)

Other top bar:

![image](https://user-images.githubusercontent.com/57354320/80861705-81cad180-8c70-11ea-83fd-a9736f993141.png)

**Comment by ludeeus** [comment]
They are not the same, they are similar, but not the same.

**Comment by chemelli74** [comment]
> They are not the same, they are similar, but not the same.

Can you be so kind to fix it ?
OR may I to a PR ?

Thank you in Advance,

Simone

**Comment by ludeeus** [comment]
Go for it :tada:

**Comment by chemelli74** [comment]
> Go for it 🎉

I just need a few directions, if possible.
Is enough to uncompressing "uilogs.js.gz", modify it, recompressing and restart HA?

Simone

**Comment by ludeeus** [comment]
Not really, it needs to be changed in https://github.com/custom-components/uilogs/blob/master/src/main.ts
Then compiled with `npm run build`

**Comment by chemelli74** [comment]
> Not really, it needs to be changed in https://github.com/custom-components/uilogs/blob/master/src/main.ts
> Then compiled with `npm run build`

Thankyou, just installed all plugin dependencies and compiled.
Tomorrow will start digging into the code.

Simone

**Comment by chemelli74** [comment]
PR #10 

Simone

**Comment by chemelli74** [comment]
@ludeeus did you get time to review my PR ? ;-)

Simone

**Comment by chemelli74** [comment]
Sorry to bother you again, but after the update I cannot see any change in my title bar :-(

Simone

**Comment by ludeeus** [comment]
You PR did not include the compiled version .gz
But it has been overwritten now to use the new HA syle anyway

**Comment by chemelli74** [comment]
> You PR did not include the compiled version .gz

Sorry, I'm not used to include binaries in PR.
I was expecting author to compile and publish new GitHub release (BT, I dont see any release in this repo).

> But it has been overwritten now to use the new HA syle anyway

Nice, let me try again then.

Simone

**Comment by chemelli74** [comment]
Works but is not positioned as I was expecting.
Also it's not respecting any more title colors:

![image](https://user-images.githubusercontent.com/57354320/81583651-ef35db00-93b1-11ea-9cc8-a50d04b9fee6.png)

Another windows is looking like this:

![image](https://user-images.githubusercontent.com/57354320/81583695-fd83f700-93b1-11ea-8c60-ab0b952e8d81.png)

Simone

**Comment by ludeeus** [comment]
It is using the new layout, that you find under configuration and in the supervisor panel.

**Comment by chemelli74** [comment]
> It is using the new layout, that you find under configuration and in the supervisor panel.

Configuration->Lovelace dashboards ?

I can be wrong, but is not reflecting current theme.
Let me know if I can do anything to help digging.

Simone

**Comment by ludeeus** [comment]
There is nothing to dig into, this is how it is.
![image](https://user-images.githubusercontent.com/15093472/81596897-1b0e8c00-93c5-11ea-9f0a-c86b42a8151b.png)
![image](https://user-images.githubusercontent.com/15093472/81596906-1e097c80-93c5-11ea-8010-94208164850a.png)
![image](https://user-images.githubusercontent.com/15093472/81596911-1fd34000-93c5-11ea-9c4a-23fef9e1ac35.png)
![image](https://user-images.githubusercontent.com/15093472/81596915-219d0380-93c5-11ea-8995-5c8e6bc54ee8.png)

**Comment by chemelli74** [comment]
> There is nothing to dig into, this is how it is.
> ![image](https://user-images.githubusercontent.com/15093472/81596897-1b0e8c00-93c5-11ea-9f0a-c86b42a8151b.png)

The title is not respecting the theme.
You can see that the background is white, while in all the other pages (map, history, logbook, etc.) is light blue.
Also font size is a lot smaller that all the other page.

Simone

**Comment by ludeeus** [comment]
> The title is not respecting the theme.

Respectfully, you are wrong.
This integration now uses the **NEW** layout.
All pages in home assistant, has not _yet_ migrated.
This is the last I will comment on this matter.
